### DISCUSSION TITLE
Best Practice for ingredients without units

### DISCUSSION THREAD

**Discussion by reibuehl** [TandoorRecipes/recipes#1534] [General]
I was wondering how to best enter ingredients that normally don't have a unit- e.g. 1 egg. Do I need to create a "piece" unit or is there a better way?

**Comment by smilerz** [comment]
you can leave the field blank (which is normally what I do) or you can do a generic (whole, piece) or a size (small, medium, large)

**Reply by reibuehl** [reply]
> you can leave the field blank

How? I wasn't able to do that. I always get "g" as this is set under Settings->Preferences->Default Unit and the Default Unit is a mandatory field that can't be left blank.

**Reply by smilerz** [reply]
The interface isn't very intuitive and will be rewritten - but for now you can Backspace in the field and it will clear it.  The drop down should also have a 'Remove' label on the selected unit.
